Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6-alpine
+
+COPY requirements.txt /app/
+
+RUN true \
+	&& pip install gunicorn \
```

Comment:
I see it as a dependency of the image rather of the package.
It is not imported anywhere in the Python code. To run the app using gunicorn is a decision on the image level, it could be accomplished otherwise (i.e. using just the python interpreter).